HIP/hip-904-frictionless-airdrop.md

```diff
@@ -0,0 +1,706 @@
+- If an account has a one or more `maxAutomaticAssociations` slots, then we can infer that the account holder is willing/interested in receiving some unsolicited tokens in their account. Therefore, the network will leave this setting to be as it is.
+- If an account has an account with `maxAutomaticAssociations` set to max integer value, then we can infer that the account holder is willing/interested in receiving unlimited unsolicited tokens in their account. Therefore, the network will change the value of `maxAutomaticAssociations` to -1.
+
+We encourage the community to weigh in on this proposal.
+
+### Fees and Payments
+
+Any software attempting to pay for auto association slots will now be charged less (i.e. they will not be charged for auto association slots). This should work, although may be surprising.
+
+Any software attempting to send an unassociated token to an account with free auto-association slots will now be charged more than they had been previously, and those transactions may now fail.
+
+## Security Implications
+
+- **Node security** Since this HIP still maintains the concept of token-associations at the ledger level and since somebody (initially the sender and later the recipient) always pays for the resources used by the token-associations, there is no additional security risk to the nodes.
+- **User security** Similar to most other layer-1 networks, any newly created account can now receive unsolicited airdrop tokens. Scammers typically use these as a vehicle to defraud innocent users. Account holders will need to be vigilant about the tokens that show up in their account, and `reject` unwanted tokens as necessary. Wallets can help by providing useful UI for the users to know what tokens have been airdropped to the user, and provide a useful way to reject some or all of them.
+
+## How to Teach This
+
+For a HIP that adds new functionality or changes interface behaviors, it is helpful to include a section on how to teach users, new and experienced, how to apply the HIP to their work.
+
+## Reference Implementation
+
+The reference implementation must be complete before any HIP is given the status of “Final”. The final implementation must include test code and documentation.
+
+## Rejected Ideas
+
+Throughout the discussion of a HIP, various ideas will be proposed which are not accepted. Those rejected ideas should be recorded along with the reasoning as to why they were rejected. This both helps record the thought process behind the final version of the HIP as well as preventing people from bringing up the same rejected idea again in subsequent discussions.
+
+In a way, this section can be thought of as a breakout section of the Rationale section that focuses specifically on why certain ideas were not ultimately pursued.
+
+## Open Issues
+
+- Should we expose pending transfers through the RPC utilizing custom endpoints?
```

Comment:
By "RPC" here are you talking about mirror node REST endpoints?  Or a gRPC query to the consensus node?  I assume the former?  

And isn't this necessary for general user stories 6-8 that talk about showing a user or DApp pending transfers?

Reply:
Neither actually.
I meant the JSON RPC on the relay.
Let me update this